CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -25,7 +25,7 @@ <h2 class="definition-title definition-title--search-result">
                             <img class="definition-title__question-mark"
                                  src="{% static 'CreeDictionary/images/fa/question-circle-solid.svg' %}"
                                  alt="linguistic breakdown" data-cy="question-mark">
-                            <span class="definition-title__tooltip-text" data-cy="linguistic-breakdown">{% if result.linguistic_breakdown_head %}{{ result.linguistic_breakdown_head|join:" + " }} + {% endif %}<strong>{{ result.lemma_wordform.text }}-</strong> + {{ result.linguistic_breakdown_tail|join:" + " }}</span>
+                            <span class="definition-title__tooltip-text" data-cy="linguistic-breakdown">{% if result.linguistic_breakdown_head %}{{ result.linguistic_breakdown_head|join:" + " }} + {% endif %}<strong>{{ result.lemma_wordform.text }}</strong> + {{ result.linguistic_breakdown_tail|join:" + " }}</span>
```

Comment:
Good! We're showing _just_ the lemma now, instead of pretending it's the stem. @aarppe will be happy.